doc/classes/InstancePlaceholder.xml

```diff
@@ -18,13 +18,7 @@
 			<argument index="1" name="custom_scene" type="PackedScene" default="null">
 			</argument>
 			<description>
-			</description>
-		</method>
-		<method name="get_instance_path" qualifiers="const">
-			<return type="String">
-			</return>
-			<description>
-				Gets the path to the [PackedScene] resource file that is loaded by default when calling [method replace_by_instance].
+				Returns an instance of the scene handed as an argument, or the scene at [member instance_path] if no argument is given. As for all resources, the scene is loaded only if it's not loaded already. By manually loading the scene beforehand, delays caused by this function can be avoided. If [replace] is [true], replaces this placeholder by the created instance.
```

Comment:
`[code]replace[/code] is [code]true[/code]`
The `[Name]` markup is only for classes, to make a reference to the class.